src/Form/Select.js

```diff
@@ -7,23 +7,23 @@ import Icon from '../Icon';
 import Flex from '../Flex';
 import Label from './Label';
 import { createEasyInput } from './EasyInput';
-import { createComponent } from '../utils';
+import { themeGet, createComponent } from '../utils';
 
 const SelectContainer = createComponent({
   name: 'SelectContainer',
   as: Flex,
-  style: ({ size, theme, value, borderRadius = theme.radius }) => css`
+  style: ({ theme, value }) => css`
     background: white;
     border: 1px solid ${theme.colors.greyLight};
-    height: ${theme.heights[size]}px;
+    height: 48px;
```

Comment:
Do we want to reuse variables to hardcode this size via `themeGet('heights.xl')px;` here and for other size magic string above (e.g. font-size), even though they are no longer configurable?

Reply:
No - I want to move away from heights in totality. It's too brittle.